src/main/resources/logback.xml

```diff
@@ -1,25 +1,25 @@
 <configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+            <pattern>%d{HH:mm:ss.SSS} %-5level - %msg%n</pattern>
```

Comment:
I would keep logger, as some modules have their own logger

Reply:
The only module that uses its own logger currently is `LanguageModule`. Do you think it makes sense to add loggers to other modules too? I'd have gone the route of modifying the ModuleResponse to return arbitrary messages from the modules instead of having an extra logger for each module

Reply:
I think it's cleaner if modules log themselves instead of sending that over somewhere

Reply:
Hmm, yeah after thinking about it it's probably best to keep thread and logger there, just in case for future expansion, and it also doesn't really hurt to have them there.

I have an idea on how to improve module feedback and logging, but I'll first need to experiment a bit to see if it works